crates/nu-command/tests/commands/reject.rs

```diff
@@ -185,3 +185,9 @@ fn reject_multiple_rows_descending() {
     let actual = nu!("[[a,b];[1 2] [3 4] [5 6]] | reject 2 1 | to nuon");
     assert_eq!(actual.out, "[[a, b]; [1, 2]]");
 }
+
+#[test]
+fn test_ignore_errors_flag() {
+    let actual = nu!("[[a, b]; [1, 2], [3, 4], [5, 6]] | reject 5 -i | to nuon");
```

Comment:
So the main reason for `-i` is reject by variables, would you please provide a test for that case?

Reply:
Sure! I've added the test case for List of CellPaths :smile: